linting/lint-staged.config.js

```diff
@@ -5,12 +5,12 @@
 
 // lint-staged will call these scripts with a set of files - eg npm run x 'file.txt' 'styling.css' etc
 const lintChecksForAllFiles = ['npm run lint:allfiles', 'npm run lint:format']; // note: the licence tool, run as a part of `allfiles`, will update non staged files as well - this cannot be configured currently.
-const lintChecksForJSFiles = ['npm run lint:js'];
+const lintChecksForTSFiles = ['npm run lint:ts'];
```

Comment:
I think while there are still `.js` files in the repo, it is worth having (seperate) linting for both (hence my changes here: https://github.com/strimzi/strimzi-ui/pull/40/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R28-R29

Reply:
Conversely - https://github.com/strimzi/strimzi-ui/pull/46/files#diff-f7186e749448c8bd4ca0e5137155149f0228e894891558f8da62a26c23e5e321R6 - I ended up setting overrides for eslint to handle js/ts differently

Reply:
I added it back I can just take the configs from one of your PRs.